### ISSUE TITLE
add attribute days untill event start

### ISSUE THREAD

**Issue by floris-b** [tybritten/ical-sensor-homeassistant#11]
I know this can be done with a template sensor but that would add an other sensor

```
    def update(self):
        """Get the latest update and set the state and attributes."""
        # Defaults:
        self._state = "No event"
        # I guess the number and details of attributes probably
        # shouldn't change, so we should really prepopulate them.
        self._event_attributes = {
            'name': None,
            'start': None,
            'end': None,
            'eta': None
        }
        # Get the data
        self.data_object.update()

        event_list = self.data_object.data
        if event_list and (self._eventno < len(event_list)):
            val = event_list[self._eventno]
            start = val['start'].datetime
            self._event_attributes['start'] = start
            self._event_attributes['eta'] = (start - dt.datetime.now(start.tzinfo )+ timedelta(days=1)).days
            name = val.get('name', 'unknown')
            self._event_attributes['name'] = name
            self._state = "{} - {}".format(
                name,
                start.strftime("%-d %B %Y %H:%M")
            )

```

**Comment by tybritten** [comment]
I'm fine with adding that- feel free to submit a PR, and I'll get it merged pretty quick.

**Comment by Olen** [comment]
This was implemented in PR #14 yesterday.

Seems to be working fine here at least.
Nice to be able to do a simple {% if eta == 0 %} to get todays appointments.

**Comment by tybritten** [comment]
Yup, going to close this. Nice work @Olen
